fr.janssoone.pkgtransporter.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id" : "fr.janssoone.pkgtransporter",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "package-transporter",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+	"--filesystem=/var/lib/flatpak/app:ro",
+        "--filesystem=xdg-data:ro",
+        "--filesystem=xdg-config:ro",
+        "--filesystem=~/.var/app/:ro"
```

Comment:
some of these filesystem permission are not allowed by default.

Reply:
The app need access to ~/.var and /var, at least ~/.var
How can I allow theses paths ?

Reply:
lines 13-16.

xdg-data and xdg-config are not allowed by default either.

https://docs.flathub.org/docs/for-app-authors/linter

Reply:
I understand how to add exceptions for `xdg-*` directories but I can't find the linter error for custom paths.

Reply:
they are not blocked by the linter.

Reply:
Okay so I don't need exceptions for this two paths ?